app/assets/stylesheets/components/sub_nav.scss

```diff
@@ -10,21 +9,36 @@
     padding-bottom: 10px;
   }
 
+  .nav-links {
+    display: flex;
+    justify-content: space-around;
+
+    @media (min-width: 640px) {
+      justify-content: flex-end;
```

Comment:
Right align the subnav links on desktop. Centred otherwise.

Reply:
Would it be worth putting this comment as documentation instead?